presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,80 @@
+=============
+Release 0.239
+=============
+
+**Highlights**
+==============
+* Add support for ``DEFINER`` and ``INVOKER`` view security models.
+* Add support for caching Glue metastore in Hive connector.
+* Add Pinot SQL endpoint support.
+
+General Changes
```

Comment:
We may be to add a heading (e.g. `**Details**` or `**Notes**`) in parallel with `**Highlights**` so that it doesn't looks like the sections below are sub-sections of the highlights. We're also missing this with 0.238 release notes, so I'm ok with fixing this later.